docs/chaosiq-on-docker/build-a-custom-docker-image.md

```diff
@@ -0,0 +1,47 @@
+It's recommended to use the  ```chaosiq/chaostoolkit``` Docker image from Docker Hub, which is
+provided by ChaosIQ. However if you want to build a customer Docker image, so you can add your own extensions
+the following can be used as a starter for your own Dockerfile:
+
+```
+FROM python:3.8-alpine
+
+LABEL maintainer="ChaosIQ <contact@chaosiq.io>"
```

Comment:
consider removing this label, for other to creates their own image

Reply:
yes good point